src/en/components/icon/use-case.md

```diff
@@ -6,16 +6,160 @@ eleventyNavigation:
   title: Icon
   locale: en
   parent: componentsEN
-  otherNames: glyphs, graphic, symbol.
-  description: A visual cue to help users understand the context.
-  thumbnail: /images/common/components/preview-icons.svg
-  alt: Icon component shows a green check mark icon, orange triangle with an exclamation icon, blue circle  with an i for information, and an arrow.
-  state: coming-soon
+  otherNames: symbol, glyph, graphic.
+  description: A symbol that represents an action or idea.
+  thumbnail: /images/common/components/preview-icon.svg
+  alt: 'Four symbols: An envelope, a magnifying class, a phone, and an arrow pointing down towards a horizontal line.'
+  state: published
 translationKey: 'icon'
 tags: ['iconEN', 'usage']
 permalink: /en/components/icon/
 date: 'git Last Modified'
-nocrawl: true
 ---
 
-## Usage
+## Problems icons solve
+
+Use an icon to:
+
+- Clarify information with a visual reference.
+- Highlight key actions or states, like a checkmark for validation.
+- Scan content quicker and reduce cognitive load.
+
+<article class="bg-full-width bg-primary text-light pt-600 pb-300 my-600">
+  <h2 class="mt-0">Related components</h2>
+
+<a href="{{ links.text }}" class="link-light">Text</a> to display written content in a styled and formatted paragraph.
+
+<a href="{{ links.notice }}" class="link-light">Notice</a> to display a short, prominent message.
+</article>
+
+## Icon use cases
+
+### Emphasize information
+
+Choose an icon to match the information you want to emphasize.
+
+<div class="remove-empty-p">
+<gcds-grid columns="1fr" columns-tablet="1fr 2fr" align-items="start">
```

Comment:
Basing it on the design, this looks wider to me. Is it possible to make it smaller, maybe by changing it to `1fr 4fr`?
```suggestion
<gcds-grid columns="1fr" columns-tablet="1fr 4fr" align-items="start">
```

Reply:
I made this section larger on purpose to avoid having the icon preview for the chevron section wrap to multiple lines for some smaller tablet and mobile screens.